renovate.json

```diff
@@ -7,6 +7,33 @@
     {
       "matchDepTypes": ["require"],
       "rangeStrategy": "bump"
+    },
+    {
+      "matchPackageNames": ["php"],
+      "rangeStrategy": "replace"
+    }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["^.platform.app.yaml$"],
+      "matchStrings": [" OPENSWOOLE_VERSION=(?<currentValue>.*)$"],
+      "depNameTemplate": "openswoole/ext-openswoole",
+      "datasourceTemplate": "github-releases",
+      "extractVersion": "^v(?<version>.*)$",
+      "prBodyNotes": [
+        ":warning: CI does not verify if extension can be built successfully or compatible. Build log review at platform.sh is required."
+      ]
+    },
+    {
+      "fileMatch": ["^.platform.app.yaml$"],
+      "matchStrings": ["^type: php:(?<currentValue>.*)$"],
```

Comment:
I am not sure if renovate does multiline regex matching or not.

Reply:
>The regex manager matches are done per-file and not per-line, you should be aware when using the ^ and/or $ regex assertions.
from https://docs.renovatebot.com/modules/manager/regex/